CHANGELOG.md

```diff
@@ -277,6 +277,7 @@ Other minor additions
 * Added new proofs to `Data.List.Properties`:
   ```agda
   reverse-injective : reverse xs ≡ reverse ys → xs ≡ ys
+  map-injective : ∀ {f : A → B} → Injective _≡_ _≡_ f → Injective _≡_ _≡_ (map f)
```

Comment:
You can drop the `∀ {f : A → B} → ` from the CHANGELOG.
We usually only bother listing the interesting part of a type.